portfolio/src/main/webapp/index.html

```diff
@@ -24,18 +25,20 @@
         <li><a href="about.html" style="color:white"> ABOUT </a></li>
         <li><a href="contact.html" style="color:white"> CONTACT </a></li>
         <li><a href="stock.html" style="color:white"> STOCK PICKER</a></li>
+        <li><a href="comments.html" style="color:white"> COMMENTS</a></li>
       </ul>
       <div class="button">
-        <a href="index.html">
+        <a href="index.html"></a>
         <input type="image" alt="Olu" src="images/headshot.png" style="max-width: 600px;max-height: 300px;"> 
       </div>
       <h1><i>Welcome to Olu's Portfolio</h1>
       <div class="button">
-        <a href="https://www.linkedin.com/in/olua/" target="_blank">
+        <a href="https://www.linkedin.com/in/olua/" target="_blank"></a>
         <input type="image" alt="LinkedIn" src="images/LI.png">
-        <a href="https://github.com/olu144" target="_blank">
+        <a href="https://github.com/olu144" target="_blank"></a>
         <input type="image" alt="GitHub" src="images/GH.png">		
       </div>
+      </div>

```

Comment:
Do we need this `</div>`? It seems like it could be an extra, but I might also just have missed where the opening tag is.

Reply:
No (Addressed)